src/Defs.h

```diff
@@ -147,6 +147,13 @@ extern const int MAX_FDS_NUM;
 #define DEFAULT_SELECT_TIMEOUT_MSEC	10
 #define DEFAULT_DEBUG_LEVEL			0
 
+/*
+Used by offload libraries to do egress path warm-up of caches.
+It is not in use by kernel. WARNING: it will actually end this packet on the wire.
+More info at vma_extra.h
+*/
+#define DUMMY_SEND_FLAG MSG_SYN // equals to 0x400
```

Comment:
it can be safer to add a comment - this value is shared with vma (and solarflare); hence, don't change here without changing accordingly in VMA